TeaDiary/Scripts/App/TeaCommands.js

```diff
@@ -0,0 +1,46 @@
+﻿function TeaCommands() {
```

Comment:
It's ok for now, but be mindful about such classes - they tend to become God classes when the number of supported operations increases. Essentially you will be doing the Facade pattern here, read about it.